Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -203,6 +203,12 @@ struct Scan {
     u8 successive_approximation {};
 
     HuffmanStreamState huffman_stream;
+
+    // See the note under the first paragraph
```

Comment:
Which first paragraph? What if someone adds a paragraph somewhere? Reword this so it's clear what this refers to.